backend/de.metas.acct.base/src/main/java/de/metas/acct/vatcode/VATCode.java

```diff
@@ -34,23 +35,28 @@
 {
 	public static final VATCode NULL = new VATCode();
 
-	public static VATCode of(final String code)
+	public static VATCode of(final String code, final int vatCodeId)
```

Comment:
boy scout rule-ish: could u pls remove `VATCode.NULL`

and change de.metas.acct.vatcode.impl.VATCodeDAO#findVATCode to return Optional<VATCode>

?

Reply:
done.